### DISCUSSION TITLE
v-carousel issue

### DISCUSSION THREAD

**Discussion by manishrajmishra** [vuetifyjs/vuetify#13782] [Q&A] [answered]
While using v-carousel, in network tab why we are getting two request for the same image with different initiator  VImg.ts and (index). Can any help me understanding this ?

**Comment by KaelWD** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/16421948/121489022-feed7c00-ca16-11eb-8019-eac804501ed6.png)
